Libraries/Components/Touchable/TouchableWithoutFeedback.js

```diff
@@ -253,25 +255,48 @@ class TouchableWithoutFeedback extends React.Component<Props, State> {
 
 function createPressabilityConfig(props: Props): PressabilityConfig {
   return {
-    cancelable: !props.rejectResponderTermination,
```

Comment:
these props can be deleted?

Reply:
These props aren't part of `PressabilityConfig` in 0.62-stable. They also make flow sad, so we don't really have much of a choice.